<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingWindow</class>
 <widget class="QMainWindow" name="WorkingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/icons/icons/simpleASM.png</normaloff>:/new/icons/icons/simpleASM.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QTextBrowser" name="output">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Output will be displayed here&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-----------------------------------------&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Search for a command</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="commandSearch">
         <property name="placeholderText">
          <string>Write command and press enter ... </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="commandInfo">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Short info about commands&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;--------------------------------------&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save (Ctrl+S)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/save.png</normaloff>:/new/icons/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buildButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="toolTip">
          <string>Build and Link (Ctrl+B)
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/build.png</normaloff>:/new/icons/icons/build.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="executeButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Execute (Ctrl+R)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/run.png</normaloff>:/new/icons/icons/run.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="debugButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Debug (Ctrl + D)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/debug.png</normaloff>:/new/icons/icons/debug.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="infoButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Info (Ctrl+I)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/info.png</normaloff>:/new/icons/icons/info.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Exit (Alt+x)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/new/icons/icons/exit.png</normaloff>:/new/icons/icons/exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="input">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(212, 68, 34);
image: url(:/new/icons/icons/simpleASM.png);
</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>widget</zorder>
   <zorder>output</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>input</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build and Link</string>
    </property>
    <addaction name="actionBuild_and_Link"/>
   </widget>
   <widget class="QMenu" name="menu_Run">
    <property name="title">
     <string>&amp;Run or Debug</string>
    </property>
    <addaction name="action_Execute"/>
    <addaction name="action_Debug"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
   <addaction name="menu_Run"/>
   <addaction name="menu_About"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionBuild_and_Link">
   <property name="text">
    <string>Build and Link</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_Execute">
   <property name="text">
    <string>&amp;Execute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Debug">
   <property name="text">
    <string>&amp;Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>saveButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBuild_and_Link</sender>
   <signal>triggered()</signal>
   <receiver>buildButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Execute</sender>
   <signal>triggered()</signal>
   <receiver>executeButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Debug</sender>
   <signal>triggered()</signal>
   <receiver>debugButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>696</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInfo</sender>
   <signal>triggered()</signal>
   <receiver>infoButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>exitButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>904</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
